alfresco-java-rest-api/alfresco-java-rest-api-lib/alfresco-event-gateway-api/src/main/java/org/alfresco/gateway/model/JsonNode.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021-2021 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.alfresco.gateway.model;
+
+import java.util.Objects;
+import org.springframework.validation.annotation.Validated;
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+
+/**
+ * JsonNode
+ */
+@Validated
+public class JsonNode   {
```

Comment:
Is this class really needed? It has no attributes, nor method (beyond those overridden from Object)

Reply:
True, not needed, the codegen took it as a custom domain model